<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RParametricAnalysisDialogUi</class>
 <widget class="QWidget" name="RParametricAnalysisDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>483</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Parametric Analysis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="parametersGroupBox">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="parametersNameLabel">
          <property name="text">
           <string>Parameter Names</string>
          </property>
          <property name="buddy">
           <cstring>parametersLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="minValuesLabel">
          <property name="text">
           <string>Minimum Values</string>
          </property>
          <property name="buddy">
           <cstring>minLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="maxValuesLabel">
          <property name="text">
           <string>Maximum Values</string>
          </property>
          <property name="buddy">
           <cstring>maxLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="numValuesLabel">
          <property name="text">
           <string>Number of Values</string>
          </property>
          <property name="buddy">
           <cstring>countLineEdit</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="parametersLineEdit">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="minLineEdit">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maxLineEdit">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="countLineEdit">
          <property name="minimumSize">
           <size>
            <width>350</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="parameterGenerationLabel">
          <property name="text">
           <string>Parameter realizations</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="gridRadioButton">
          <property name="text">
           <string>Grid</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="randomRadioButton">
          <property name="text">
           <string>Random</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="randomGridRadioButton">
          <property name="text">
           <string>Random within grid</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="functionGroupBox">
     <property name="title">
      <string>Function Evaluation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Function Names</string>
          </property>
          <property name="buddy">
           <cstring>functionsLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="functionsLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QCheckBox" name="plot2DCheckBox">
        <property name="text">
         <string>Plot</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="printCheckBox">
        <property name="text">
         <string>Print to the output pane</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="outputFileLabel">
          <property name="text">
           <string>Output File</string>
          </property>
          <property name="buddy">
           <cstring>outputFileLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="outputFileLineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="browseButton">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>25</height>
           </size>
          </property>
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="analyzerGroupBox">
     <property name="title">
      <string>Other Analysis Types</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QLabel" name="analyzerLabel">
        <property name="text">
         <string>Choose an Analyzer from the list</string>
        </property>
        <property name="buddy">
         <cstring>analyzerComboBox</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="analyzerComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>parametersLineEdit</tabstop>
  <tabstop>minLineEdit</tabstop>
  <tabstop>maxLineEdit</tabstop>
  <tabstop>countLineEdit</tabstop>
  <tabstop>gridRadioButton</tabstop>
  <tabstop>randomRadioButton</tabstop>
  <tabstop>randomGridRadioButton</tabstop>
  <tabstop>functionGroupBox</tabstop>
  <tabstop>functionsLineEdit</tabstop>
  <tabstop>plot2DCheckBox</tabstop>
  <tabstop>printCheckBox</tabstop>
  <tabstop>outputFileLineEdit</tabstop>
  <tabstop>browseButton</tabstop>
  <tabstop>analyzerGroupBox</tabstop>
  <tabstop>analyzerComboBox</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
